ynr/apps/parties/management/commands/parties_import_from_ec.py

```diff
@@ -61,6 +68,26 @@ def handle(self, *args, **options):
             )
             for party in importer.collector:
                 self.stdout.write(str(party))
+
+            if options["post_to_slack"]:
```

Comment:
Do we need a special flag for this, or would it be reasonable to just test for `if settings.SLACK_TOKEN`?

Reply:
I'd like this as a flag. Because this is a management command that could be used in a variety of ways, just looking at the API token isn't enough. Examples: 

* A new install runs this to set things up, happens to have a valid token
* For some reason we have to drop and re-import all parties one day (the EC updates thrie ID system, or something)
* Most likely: I screw up and forget I have a token in my `local.py` file, do something that spams `#candidates`.

Because of this, off by default seems like a good idea.

Reply:
I also don't _really_ care if `post_to_slack` is on and the token isn't set – it will be obvious what the problem is, and catching the error doesn't add a lot

Reply:
Yeah I see what you mean. Its a slightly different case to [this one](https://github.com/DemocracyClub/WhoCanIVoteFor/blob/master/wcivf/apps/feedback/management/commands/batch_feedback_to_slack.py) or [this one](https://github.com/DemocracyClub/EveryElection/blob/master/every_election/apps/election_snooper/helpers.py), although if you're routinely running your local dev copy with `SLACK_WEBHOOK_URL` set, there are plenty of other foot-guns there for you :grimacing:

Reply:
I am not doing that, but my ability to screw up is limitless